app-config.yaml

```diff
@@ -44,10 +44,10 @@ techdocs:
     techdocs: 'docker'
 
 sentry:
-  organization: spotify
+  organization: My Company
 
 rollbar:
-  organization: spotify
+  organization: My Company
```

Comment:
This and the one above are actually meant to be identifiers right? So my-company could be a better example perhaps?